packages/SwingSet/src/vats/vat-timerWrapper.js

```diff
@@ -41,6 +42,29 @@ export function buildRootObject(vatPowers) {
         repeaters.set(index, vatRepeater);
         return vatRepeater;
       },
+      createNotifier(delaySecs, interval) {
```

Comment:
Is the type of this declared somewhere? Is there a doc-comment explaining what it does? Any tests?

Reply:
Is it ok that, once created, it lasts and remains active forever? If not, how about returning a kit of a notifier and a cancel object?